static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -119,10 +121,11 @@ const isCompatibleWithEpic = (page: Object): boolean =>
     isArticleWorthAnEpicImpression(page, defaultExclusionRules);
 
 const shouldShowReaderRevenue = (
```

Comment:
I had a thought about making this clearer! what about splitting it into `shouldShowToSupporters()` and `shouldShowToNonSupporters()`, duplicating the `!config.get('page.shouldHideReaderRevenue')`. And then just calling the appropriate function rather than passing a mysterious boolean at the call site

Reply:
💯